kfp/doc/setup.md

```diff
@@ -66,10 +66,11 @@ choose your OS system, and process according to "(Optional) Install the MinIO Cl
 
 ## Installation steps <a name = "installation"></a>
 
-You can create a Kind cluster with all required software installed using the following command: 
+You can create a Kind cluster with all required software installed using the following command, run in the `kind` directory under the repo root: 
```

Comment:
Hi @deanwampler , sorry maybe it is my English, but the sentence is not clear to me especially that the first command is `cd kind`. 
Will it be better:
You can create a Kind cluster with all required software installed using the following command:
```shell
make -C kind setup
```

Reply:
Maybe I missed that. Sorry.

Reply:
Pushing an update...